src/platform/packages/private/kbn-reporting/get_csv_panel_actions/panel_actions/get_csv_panel_action.tsx

```diff
@@ -82,15 +82,20 @@ interface ExecutionParams {
   i18nStart: I18nStart;
 }
 
-type GetCsvActionApi = HasType & PublishesSavedSearch & CanAccessViewMode & HasTimeRange;
+type GetCsvActionApi = HasType &
+  PublishesSavedSearch &
+  CanAccessViewMode &
+  HasTimeRange &
+  PublishesTitle;
 
 const compatibilityCheck = (api: EmbeddableApiContext['embeddable']): api is GetCsvActionApi => {
   return (
     apiHasType(api) &&
     apiIsOfType(api, SEARCH_EMBEDDABLE_TYPE) &&
     apiPublishesSavedSearch(api) &&
     apiCanAccessViewMode(api) &&
-    Boolean((api as unknown as HasTimeRange).hasTimeRange)
+    Boolean((api as unknown as HasTimeRange).hasTimeRange) &&
+    Boolean((api as unknown as PublishesTitle).title$)
```

Comment:
```suggestion
    apiPublishesTitle(api)
```
I think we can use the `apiPublishesTitle` guard for this one.

Reply:
applied the change here : 3ecb6bf999099ec036f21655070491d0d609b886